plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -30,6 +36,14 @@ def build
     @image.remove(force: true) if @image
   end
 
+  def run_pre_build_script
+    file = File.join(@dir, PRE_BUILD_SCRIPT)
+    return unless File.exist?(file)
```

Comment:
nitpick: you can use [`File.file?`](http://ruby-doc.org/core-2.2.0/File.html#method-c-file-3F) to be a bit more specific